### PR TITLE
fix: image on the edit welcome post is shouldn't be rounded

### PR DESCRIPTION
**Pull request by TomDowling**
## Changes
Removed the prop to make the image circular. I didn't find anywhere affected from removing this prop

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1361 #done

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]

**Review by sshanzel — CHANGES_REQUESTED** [review]
We actually have to keep it depending on the type of comment box. This is called `ProfilePicture` but unfortunately being used by `Source` entity on some cases.

File: packages/shared/src/components/modals/CommentBox.tsx [hunk 1]
```diff
@@ -140,12 +140,20 @@ function CommentBox({
...
-          <ProfilePicture
-            size="large"
-            rounded="full"
-            user={{ image: authorImage, username: authorName }}
-            nativeLazyLoading
-          />
+          {isComment ? (
+            <ProfilePicture
+              size="large"
+              rounded="full"
```
Comment (rebelchris) [inline packages/shared/src/components/modals/CommentBox.tsx:146]: You can just make this line optional based on isComment, but is it the right evaluation? Not sure what it fires on.

File: packages/shared/src/components/modals/CommentBox.tsx [hunk 2]
```diff
@@ -140,12 +140,20 @@ function CommentBox({
...
+              user={{ image: authorImage, username: authorName }}
+              nativeLazyLoading
+            />
+          ) : (
+            <ProfilePicture
+              size="large"
+              user={{ image: authorImage, username: authorName }}
+              nativeLazyLoading
+            />
+          )}
```
Comment (sshanzel) [inline packages/shared/src/components/modals/CommentBox.tsx:156]: I think their only difference is the property. I think we can just apply the condition from the property itself. WDYT?
```suggestion
          <ProfilePicture
            size="large"
            rounded={isComment && 'full'}
            user={{ image: authorImage, username: authorName }}
            nativeLazyLoading
          />
```

**Review by sshanzel — CHANGES_REQUESTED** [review]
Just one last thing 🥳

**Review by sshanzel — APPROVED** [review]
Way better with `undefined`! 🚀 🚀 🚀

**Review by rebelchris — APPROVED** [review]
Love it 🤙

**Pull request merged**